src/main/java/uk/gov/dvsa/mot/fixtures/generic/WebStepDefinitions.java

```diff
@@ -33,6 +33,9 @@ public WebStepDefinitions(WebDriverWrapper driverWrapper) {
         When("^I press the \"([^\"]+)\" button$", (String buttonText) ->
                 driverWrapper.pressButton(buttonText));
 
+        When("^I press the first \"([^\"]+)\" button$", (String buttonText) ->
```

Comment:
Please add this new step definition to the docs

mot-automated-testsuite/docs/step-definitions.md

Reply:
Added.